pyproject.toml

```diff
@@ -0,0 +1,5 @@
+[build-system]
+requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4"]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
```

Comment:
Sorry for my ignorance, but is this line needed, and why? (I accept a link with a 2-lines explanataion).

Reply:
This is the line which causes `setuptools_scm` to be enabled ([docs](https://github.com/pypa/setuptools_scm/#pyprojecttoml-usage)) so it automatically extracts the version metadata from the git history instead of requiring it to be hard coded.

Reply:
just add that comment of yours as a comment in `pyporject.toml`, so no one ever asks anymore :D